yarn.lock

```diff
@@ -233,10 +233,10 @@
     "@types/estree" "*"
 
 "@types/body-parser@*":
-  version "1.16.8"
-  resolved "https://registry.npmjs.org/@types/body-parser/-/body-parser-1.16.8.tgz#687ec34140624a3bec2b1a8ea9268478ae8f3be3"
+  version "1.17.0"
```

Comment:
uh, wow...  there was a lot of churn in this file.  All of that is from adding chai-as-promised, right?

Reply:
This took all morning to resolve, but I am now happy to report that the diff is now as expected.